File: 1-iniciante/1010/1010.c
#include <stdio.h>

int main() {
    int part1, part2, qty1, qty2;
    double cost1, cost2;
    scanf("%d%d%lf", &part1, &qty1, &cost1);
    scanf("%d%d%lf", &part2, &qty2, &cost2);
    printf("VALOR A PAGAR: R$ %.2lf\n", (qty1 * cost1) + (qty2 * cost2));
    return 0;
}

File: 1-iniciante/1010/1010.lua
part1, qty1, cost1 = io.read("*n", "*n", "*n")
part2, qty2, cost2 = io.read("*n", "*n", "*n")
print("VALOR A PAGAR: R$ " .. string.format("%.2f", (qty1 * cost1) + (qty2 * cost2)))